apps/viewdir/viewdir.yml

```diff
@@ -0,0 +1,16 @@
+name: Viewdir
+description: 'Secure image viewer featuring single hand operations.'
+website: 'https://github.com/sprout2000/viewdir'
+repository: 'https://github.com/sprout2000/viewdir'
```

Comment:
Please correct the application, what do you want to add `Viewdir` or `Less View`?

Reply:
I apologize for my failure.
I meant to want to add 'LessView' but I lost forked repo on my github account by accident.
so I can't push 'lessview.yml' any more.
Would you mind that closing this request and I will send another pull request?

Reply:
[screenshot](https://user-images.githubusercontent.com/52094761/78979723-611ec880-7b57-11ea-8789-fc91e928ade6.png)